CMakeLists.txt

```diff
@@ -827,6 +835,22 @@ set(SOURCES
         utilities/ttl/db_ttl_impl.cc
         utilities/write_batch_with_index/write_batch_with_index.cc
         utilities/write_batch_with_index/write_batch_with_index_internal.cc
+        cloud/aws/aws_env.cc
```

Comment:
should we include these files only "if AWS" is specified?

Reply:
We do not include them conditionally in the Makefile, so I think the same rule works here.  If you do not compile with AWS enabled, you will get an error attempting to use it.